src/legacy/core_plugins/kibana/public/dashboard/dashboard_state_manager.js

```diff
@@ -238,9 +238,9 @@ export class DashboardStateManager {
 
     _.forEach(getEmbeddables(store.getState()), (embeddable, panelId) => {
       if (embeddable.initialized && !this.panelIndexPatternMapping.hasOwnProperty(panelId)) {
-        const indexPattern = getEmbeddableMetadata(store.getState(), panelId).indexPattern;
-        if (indexPattern) {
-          this.panelIndexPatternMapping[panelId] = indexPattern;
+        const embeddableMetadata = getEmbeddableMetadata(store.getState(), panelId);
+        if (embeddableMetadata.indexPatterns) {
```

Comment:
Could getEmbeddableMetadata return null (even though this wasn't handled in the previous version)?

Reply:
No, metadata is defaulted to `{}` if nothing is provided.

https://github.com/elastic/kibana/blob/master/src/legacy/ui/public/embeddable/embeddable.ts#L52